tests/text/test_tabstops.py

```diff
@@ -50,17 +55,28 @@ def alignment_fixture(self, request):
         ('w:tab{w:leader=none}', 'SPACES'),
         ('w:tab{w:leader=dot}',  'DOTS'),
     ])
-    def leader_fixture(self, request):
+    def leader_get_fixture(self, request):
         tab_stop_cxml, member = request.param
         tab_stop = TabStop(element(tab_stop_cxml))
         expected_value = getattr(WD_TAB_LEADER, member)
         return tab_stop, expected_value
 
     @pytest.fixture
-    def position_fixture(self, request):
+    def position_get_fixture(self, request):
         tab_stop = TabStop(element('w:tab{w:pos=720}'))
         return tab_stop, Twips(720)
 
+    @pytest.fixture(params=[
+        ('w:tab',            Twips(720),  'w:tab{w:pos=720}'),
+        ('w:tab{w:pos=360}', Twips(720),  'w:tab{w:pos=720}'),
```

Comment:
I removed the first case here because it's invalid XML, so not a distinct case we need to concern ourselves with. This point is a little subtle, as you'll see in another comment here. It's *not* that we never provide invalid XML as the "before" XML for a case. It's just that this first case is not meaningfully distinct from the second case.